### PR TITLE
Linked all translated readme to main and updated Translations.md

### PR DESCRIPTION
**Pull request by TheCSGuy25**
Hey,

I've added shields to the main readme as well as to the Chinese, Turkish and Bengali translated readme. I also updated the Translations.md.

Hope it's ok !

### CODE REVIEW

File: translations/README.bn.md [hunk 1]
```diff
@@ -8,6 +8,10 @@
 [![GitHub forks](https://img.shields.io/github/forks/microsoft/IoT-For-Beginners.svg?style=social&label=Fork)](https://GitHub.com/microsoft/IoT-For-Beginners/network/)
 [![GitHub stars](https://img.shields.io/github/stars/microsoft/IoT-For-Beginners.svg?style=social&label=Star)](https://GitHub.com/microsoft/IoT-For-Beginners/stargazers/)
 
+[![English](https://img.shields.io/badge/-English-red)](README.md)
```
Comment (Pandz18) [inline translations/README.bn.md:11]: Is the path to the English file correct?

I clicked on copy path and I got this

**Review by Pandz18 — COMMENTED** [review]
Could you confirm that the path to the main readme is correct?

File: TRANSLATIONS.md [hunk 2]
```diff
@@ -8,6 +8,12 @@ There are [**translations**](https://github.com/microsoft/IoT-For-Beginners/tree
 
 Translated lessons should follow this naming convention:
 
+When translating the initial Readme , link the other translations to your translated readme and link yours to the main English Readme by using Shields as shown:
+
+<-code in markdown->
```
Comment (jimbobbennett) [inline TRANSLATIONS.md:13]: Can you remove `<-code in markdown->` please, this is not necessary

File: TRANSLATIONS.md [hunk 3]
```diff
@@ -8,6 +8,12 @@ There are [**translations**](https://github.com/microsoft/IoT-For-Beginners/tree
...
+```
```
Comment (jimbobbennett) [inline TRANSLATIONS.md:14]: Can you mark this code block as markdown so it gets correctly syntax highlighted please. Should be `markdown` after the opening backticks.

File: translations/README.bn.md (hunk 1 above)
Comment (jimbobbennett) [reply translations/README.bn.md:11]: Nope - this is in the `translations` folder, so you need to go up one level. Should be `../README.md`

File: translations/README.bn.md [hunk 4]
```diff
@@ -8,6 +8,10 @@
...
+[![Chinese](https://img.shields.io/badge/-Chinese-yellow)](translations/README.zh-cn.md)
```
Comment (jimbobbennett) [inline translations/README.bn.md:12]: This file is already in the translations folder, so the links to the translated versions don't need the path, just the file name.

File: translations/README.tr.md [hunk 5]
```diff
@@ -8,6 +8,10 @@
 [![GitHub forks](https://img.shields.io/github/forks/microsoft/IoT-For-Beginners.svg?style=social&label=Fork)](https://GitHub.com/microsoft/IoT-For-Beginners/network/)
 [![GitHub stars](https://img.shields.io/github/stars/microsoft/IoT-For-Beginners.svg?style=social&label=Sta)](https://GitHub.com/microsoft/IoT-For-Beginners/stargazers/)
 
+[![Bengali](https://img.shields.io/badge/-Bengali-blue)](translations/README.bn.md)
```
Comment (jimbobbennett) [inline translations/README.tr.md:11]: Again - translated ones from this folder, English from up one level

File: translations/README.zh-cn.md [hunk 6]
```diff
@@ -8,6 +8,10 @@
 [![GitHub forks](https://img.shields.io/github/forks/microsoft/IoT-For-Beginners.svg?style=social&label=Fork)](https://GitHub.com/microsoft/IoT-For-Beginners/network/)
 [![GitHub stars](https://img.shields.io/github/stars/microsoft/IoT-For-Beginners.svg?style=social&label=Star)](https://GitHub.com/microsoft/IoT-For-Beginners/stargazers/)
 
+[![Bengali](https://img.shields.io/badge/-Bengali-blue)](translations/README.bn.md)
```
Comment (jimbobbennett) [inline translations/README.zh-cn.md:11]: Again - translated ones from this folder, English from up one level

**Review by jimbobbennett — CHANGES_REQUESTED** [review]
It's worth checking all links in your fork. Can you open each file and click the shields to ensure they link to the right place.

**Comment by Pandz18** [comment]
> It's worth checking all links in your fork. Can you open each file and click the shields to ensure they link to the right place.

Sure , will do !

File: TRANSLATIONS.md (hunk 3 above)
Comment (Pandz18) [reply TRANSLATIONS.md:14]: > Can you mark this code block as markdown so it gets correctly syntax highlighted please. Should be `markdown` after the opening backticks.

Do you mean that after the opening backticks I have to use '>' for the code ?

File: translations/README.bn.md (hunk 4 above)
Comment (Pandz18) [reply translations/README.bn.md:12]: Oh yeah, sorry about that will fix it

File: TRANSLATIONS.md (hunk 3 above)
Comment (jimbobbennett) [reply TRANSLATIONS.md:14]: Check out the markdown syntax highlighting guide:

https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

you need to put the code language on the same line as the opening back ticks, immediately after them. So for this, you need:

````
```markdown
[![Bengali](https://img.shields.io/badge/-Bengali-blue)](translations/README.bn.md)
```
````

Comment (Pandz18) [reply TRANSLATIONS.md:14]: > Can you mark this code block as markdown so it gets correctly syntax highlighted please. Should be `markdown` after the opening backticks.

Hi, I apologise if I'm a bit behind in this. I'm a bit confused as in the preview this is what I see.

Comment (Pandz18) [reply TRANSLATIONS.md:14]: > > Can you mark this code block as markdown so it gets correctly syntax highlighted please. Should be `markdown` after the opening backticks.
> 
> Hi, I apologise if I'm a bit behind in this. I'm a bit confused as in the preview this is what I see.


Screen Shot 2021-07-23 at 11 40 18 PM

**Comment by Pandz18** [comment]
@jimbobbennett Hi, I've changed it as you've suggested and I also tried the shields for each readme and they work. I still apologise for the confusion on the translations.md

File: TRANSLATIONS.md (hunk 3 above)
Comment (jimbobbennett) [reply TRANSLATIONS.md]: I've edited the file to show what I mean

**Comment by jimbobbennett** [comment]
No need to apologies for the confusion at all! We really appreciate your contribution, and totally happy to help you learn more about markdown syntax, and help with this PR.

I've edited the file manually and committed the change so you can see what I meant. I'll merge this PR now.

Once again - thank you for your contribution!

**Review by jimbobbennett — APPROVED** [review]

**Comment by Pandz18** [comment]
Thank you for having me Jim ! I did learn more about the markdown syntax , thank you !

Yeah, I saw your change, I understand it now  😅 

It's been a pleasure.

I have a question, would you want me to create a new issue for each new thing? So say if I find grammatical or spelling errors and would like to add something, do I have to make seperate issues or just one for both ?
 
 Thanks for having me again Jim !

**Comment by Pandz18** [comment]
On a side note , I just checked out your YouTube channel and may I say it's amazing !

**Comment by jimbobbennett** [comment]
> Thank you for having me Jim ! I did learn more about the markdown syntax , thank you !
> 
> Yeah, I saw your change, I understand it now 😅

Glad it makes sense. Markdown can be a little confusing to start with, but it's very powerful.

> I have a question, would you want me to create a new issue for each new thing? So say if I find grammatical or spelling errors and would like to add something, do I have to make seperate issues or just one for both ?

I would say use your judgement. For example, if you want to fix 10 different spelling/grammar errors across 6 different files then just raise one pull request that contains all the fixes. Same as this one - one PR for all the shields.

If there are unrelated changes, such as adding new shields and fixing broken links, then best to create multiple PRs, one per type of change. It makes it easier to review.

**Pull request merged**